scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -47,12 +47,12 @@ _install_prerequisites_centos()
           "yum -y install rpmdevtools openssl openssl-devel bc numactl ocl-icd ocl-icd-devel")
 
     for cmd in "${CMDS[@]}"; do
-        echo $cmd
-        eval $cmd
+        echo "$cmd"
+        eval "$cmd"
         if [[ $? -ne 0 ]]; then
-            echo ERROR: failed to run $cmd >&2
-            echo Problem \(or disk space\)? >&2
-            echo . Verify that you have enough disk space, and run the script again. >&2
+            echo "ERROR: failed to run $cmd" >&2
+            echo "Problem \(or disk space\)?" >&2
```

Comment:
```suggestion
            echo "Problem (or disk space)?" >&2
```

Reply:
I thought I saw you apply this, but now it's unapplied again...

Also, note that there is another line like this down below.

Reply:
reverted by mistake via force push. Added changes back.